llvm/utils/mlgo-utils/tests/pytype.test

```diff
@@ -0,0 +1,4 @@
+# REQUIRES: system-linux, has-pytype
+
+# RUN: pytype %S/../mlgo
+# RUN: pytype %S/../tests
```

Comment:
don't you need a `CHECK` to make lit happy?

Reply:
That's only if we run `FileCheck`. Here we don't need to because we don't care about the command output, only the return code. If it's not zero, lit will fail the test.